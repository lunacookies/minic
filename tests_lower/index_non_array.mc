func main {
	x := 92
	a := x[50]
}
