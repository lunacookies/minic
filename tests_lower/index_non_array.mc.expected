func main
	var x i64
	var a void
	{
		set x = 92
		set a = <missing>
	}
tests_lower:27..28: error: cannot index into non-array type “i64”
tests_lower:13..20: warning: unused variable
tests_lower:22..32: warning: unused variable
